user/troubleshooting/vm-troubleshooting.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: doc
+title: Suspend/Resume Troubleshooting
+permalink: /doc/vm-troubleshooting/
+redirect_from:
+- /doc/remove-vm-manually/
+---
+
+# VM troubleshooting #
+
+## VM Kernel troubleshooting ##
```

Comment:
I think this needs clearer indication it's about non-default kernel choice described in https://www.qubes-os.org/doc/managing-vm-kernel/#using-kernel-installed-in-the-vm. Otherwise it sounds like it applies to any issues with a kernel in VM, which is not true.